### DISCUSSION TITLE
刚入门RSS，想请教一下如何订阅B站的up主动态和合集视频🙏🙏🙏

### DISCUSSION THREAD

**Discussion by WisOcean** [DIYgod/RSSHub#16476] [Q&A]
### 配置
我在云服务器自己部署了RSSHub，服务端用的freshrss，客户端用的Fluent Reader。

### 流程与结果
在https://docs.rsshub.app/zh/ 网站上找到了我需要的示例和路由：

>https://rsshub.app/bilibili/user/dynamic/2267573
>/bilibili/user/dynamic/:uid/:routeParams?

>https://rsshub.app/bilibili/user/channel/2267573/396050
>/bilibili/user/channel/:uid/:sid/:disableEmbed?

我将自建的订阅源和示例订阅源分别在服务端和应用端测试，结果如下：

| 订阅状态 | 服务端 | 应用端 |
|:--:|:--:|:--:|
| 动态示例 | 失败 | 成功 |
| 动态自建 | 失败 | 失败 |
| 合集示例 | 失败 | 成功 |
| 合集自建 | 失败 | 失败 |

服务端报错：
![PixPin_2024-08-19_12-56-41](https://github.com/user-attachments/assets/3f2e659b-cd21-4415-9fb4-ac764e4801d8)

客户端报错：
![PixPin_2024-08-19_12-55-23](https://github.com/user-attachments/assets/23886d68-484b-4e19-8ef7-6ac686cd6fef)

两端都没有改动其他设置，订阅豆瓣或者新闻的示例源和自建源成功。

### 求助
在此情况下该如何订阅B站的up主动态和合集视频？(在服务端方便同步)

是否有更好的思路订阅两者？

自己折腾了好久，还望解惑，感激不尽🙏🙏😭😭

**Comment by dzx-dzx** [comment]
输出日志有吗？

也许可以看看隔壁：https://github.com/DIYgod/RSSHub/discussions/16406

**Reply by WisOcean** [reply]
我看不懂隔壁 #16406 的内容，不知道怎么弄。

freshrss的日志内容应该是这个：

>[Mon, 19 Aug 2024 11:26:13 +0000] [warning] --- cURL error 28: Connection timed out after 20001 milliseconds [https://rsshub.app/bilibili/user/dynamic/2267573]

>[Mon, 19 Aug 2024 11:36:47 +0000] [warning] --- cURL error 22: The requested URL returned error: 503 [http://112.xxx.xx.195:xxxx/bilibili/user/dynamic/2267573]

文件夹里没找到fluent reader的日志内容，只有用户界面提示信息：

>Error: 503 Service Unavailable
>这是自建订阅时的日志

**Reply by dzx-dzx** [reply]
我指rsshub的日志。

**Reply by WisOcean** [reply]
是这个吧

```
info: <-- GET /bilibili/user/dynamic/2267573
error: Error in /bilibili/user/dynamic/2267573: Error: Could not find Chrome (ver. 123.0.6312.86). This can occur if either
 1. you did not perform an installation before running the script (e.g. `npx puppeteer browsers install chrome`) or
 2. your cache path is incorrectly configured (which is: /app/node_modules/.cache/puppeteer).
For (2), check out our guide on configuring puppeteer at https://pptr.dev/guides/configuration.
info: --> GET /bilibili/user/dynamic/2267573 503 19ms
```

**Comment by WisOcean** [comment]
ok，知道问题在哪里了

**Reply by yunqingqi** [reply]
问题是什么呢？

**Reply by WisOcean** [reply]
时间有点久远了，没做记录，当时应该是按照RSSHub部署页面的命令行解决了上面error中Could not find Chrome的问题。

**Reply by yunqingqi** [reply]
我把公共实例换了几个然后就可以了

**Reply by WisOcean** [reply]
你用的哪个公共实例啊，我还是发现有的up可以订阅有的不行，就很奇怪。另外你用的啥客户端呢

**Reply by yunqingqi** [reply]
> 你用的哪个公共实例啊，我还是发现有的up可以订阅有的不行，就很奇怪。另外你用的啥客户端呢

就是rss hub，提供的那些公共实例，我用的查看的客户端是fluent reader
